app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/type/bundle/option/checkbox.phtml

```diff
@@ -14,16 +14,17 @@
     </label>
     <div class="control">
         <div class="nested options-list">
-            <?php if ($block->showSingle()) : ?>
+            <?php if ($block->showSingle()): ?>
                 <?= /* @noEscape */ $block->getSelectionQtyTitlePrice($_selections[0]) ?>
                 <?= /* @noEscape */ $block->getTierPriceRenderer()->renderTierPrice($_selections[0]) ?>
                 <input type="hidden"
                        class="bundle-option-<?= $block->escapeHtmlAttr($_option->getId()) ?>  product bundle option"
                        name="bundle_option[<?= $block->escapeHtml($_option->getId()) ?>]"
                        value="<?= $block->escapeHtmlAttr($_selections[0]->getSelectionId()) ?>"/>
-            <?php else :?>
-                <?php foreach ($_selections as $_selection) : ?>
+            <?php else: ?>
+                <?php foreach ($_selections as $_selection): ?>
                     <div class="field choice">
+                        <?php /** phpcs:disable */ ?>
```

Comment:
Hi @ProkopovVitaliy. Disabling static checks allowed only for some rare cases. We cannot disable the static code checks for fixing static test :)
If there's an issue in the code style it needs to be fixed by adjusting/refactoring the problematic code.

Reply:
This template really looks scary 😅. 
Will fix it.

Reply:
@rogyar I refactored template. Also  I think failing tests are not related to the changes presented in the current PR 🤔.